actors/migration/nv4/miner.go

```diff
@@ -11,6 +11,7 @@ import (
 	cbor "github.com/ipfs/go-ipld-cbor"
 	"golang.org/x/xerrors"
 
+	builtin2 "github.com/filecoin-project/specs-actors/v3/actors/builtin"
```

Comment:
This alias and path don't match, but I've made it consistent with what's there now. We'll either fix these all in bulk or, more likely, delete the `nv4` and `nv7` migrations from actors v3.